ExtLibs/Comms/CommsUDPSerialConnect.cs

```diff
@@ -92,7 +92,11 @@ public void Open(string host, string port)
             if (IsInRange("224.0.0.0", "239.255.255.255", hostEndPoint.Address.ToString()))
             {
                 log.Info($"UdpSerialConnect bind to port {Port}");
-                client = new UdpClient(int.Parse(Port), hostEndPoint.AddressFamily);
+
+                client = new UdpClient();
+                client.ExclusiveAddressUse = false;
+                client.Client.SetSocketOption(SocketOptionLevel.Socket, SocketOptionName.ReuseAddress, true);
+                client.Connect(hostEndPoint.Address, hostEndPoint.Port);
```

Comment:
this actually removes the bind port, and changes to a connect vs a bind and listern

Reply:
Exactly. We don't want to bind. We want two apps to be able to use the same multicast ip and port. If you don't like that, I suppose we could add a checkbox to enable/allow port-reuse in the CoT menu